docs/reference/aggregations/bucket/range-field-note.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[search-aggregations-bucket-range-field-note]]
+=== Note on Bucketing Range Fields
+
+==== Documents are counted for each bucket they land in
+
+Since a range represents multiple values, running a bucket aggregation over a
+range field can result in the same document landing in multiple buckets.  This
+can lead to surprising behavior, such as the sum of bucket counts being higher
+than the number of matched documents.  For example, consider this histogram
+over an integer range from 10 to 20: 
+
+[source, console]
+--------------------------------------------------
+PUT range_index
+{
+  "settings": {
+    "number_of_shards": 2
+  },
+  "mappings": {
+    "properties": {
+      "expected_attendees": {
+        "type": "integer_range"
+      },
+      "time_frame": {
+        "type": "date_range",
+        "format": "yyyy-MM-dd HH:mm:ss||yyyy-MM-dd||epoch_millis"
+      }
+    }
+  }
+}
+
+PUT range_index/_doc/1?refresh
+{
+  "expected_attendees" : {
+    "gte" : 10,
+    "lte" : 20
+  },
+  "time_frame" : {
+    "gte" : "2019-10-28 12:00:00",
```

Comment:
Do we need the time as part of this example?  Might simplify things if we just use dates (and then the mapping can be simplified too).

Also that's my birthday :tada:

Reply:
Oh far out 🎉 .  I just copied this example from the Range Mapping doc, so it would look familiar.  We don't need times though, so I can drop that.